src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,33 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  selectedTodo: Todo,
+  changeSelectedTodo: (value: null) => void,
+};
+
+export const TodoModal: React.FC<Props> = ({
+  selectedTodo,
+  changeSelectedTodo,
+}) => {
+  const [user, setUser] = useState<User | null>(null);
+
+  const {
+    id, title, userId, completed,
+  } = selectedTodo;
+
+  useEffect(() => {
+    getUser(userId).then((foundUser) => setUser(foundUser));
+  }, [selectedTodo]);
 
-export const TodoModal: React.FC = () => {
   return (
     <div className="modal is-active" data-cy="modal">
       <div className="modal-background" />
 
-      {true ? (
+      {!user ? (
```

Comment:
Better to create `isLoading` state for handling the loader state. What if you get an empty response for the user? You will see the loader all the time

Reply:
But in this case, we cannot be sure that we will get our user. I suppose it would be a good idea, if we had some error messages in this task

Reply:
You can add some error message(design is up to you), or just print a message to the console(for now, to not waste a lot of time) and close the modal window, it will be better than just show the loader all the time